src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -52,15 +52,20 @@ abstract class Particle : WasmObject() {
     abstract fun onHandleUpdate(handle: Handle)
     abstract fun onHandleSync(handle: Handle, allSynced: Boolean)
 
-    fun renderSlot(slotName: String, sendTemplate: Boolean = true, sendModel: Boolean = true) {
-      val template = if (sendTemplate) getTemplate(slotName) else ""
-      var model = ""
-      if (sendModel) {
-        val dict = populateModel(slotName)
-        model = StringEncoder.encodeDictionary(dict)
-      }
+    fun renderOutput() {
+      log("renderOutput")
+      var slotName = ""
+      val template = getTemplate(slotName)
+      val dict = populateModel(slotName)
+      var model = StringEncoder.encodeDictionary(dict)
```

Comment:
I think these can all be `val`s.

Reply:
Doh, I didn't even notice those keywords were different. Interesting design decision for Kotlin, lol.